mmvstr.h

```diff
@@ -106,14 +106,13 @@ outside of 'let(&' assignments; for example
           print2("%s\n",left(string1,70));
 
 will allocate another 70 bytes or so of memory each pass through the loop.
-If necessary, dummy 'let(&' assignments can be made periodically to clear
+If necessary, 'freeTempAlloc' can be used periodically to clear
 this temporary memory:
 
-        for (i=0; i<10000; i++)
-          {
+        for (i=0; i<10000; i++) {
           print2("%s\n",left(string1,70));
-          let(&dummy,"");
-          }
+          freeTempAlloc();
```

Comment:
This seems to be part of a comment.  Is this semantically correct?

Reply:
Yes, this is updating the comment to stop recommending the idiom of using a dummy and call `freeTempAlloc` directly.

Reply:
The risk inherent to freeTempAlloc should be noted somewhere